infrastructure/lambda-bulk-upload-report.tf

```diff
@@ -18,6 +18,8 @@ module "bulk-upload-report-lambda" {
   }
   is_gateway_integration_needed = false
   is_invoked_from_gateway       = false
+  memory_size                   = 512
+  lambda_timeout                = 45
```

Comment:
Default is 30 seconds, is 15 seconds extra enough of a buffer to warrant this change or should we increase further?

Reply:
Will confirm this in huddle